swarm-private/values.yaml

```diff
@@ -63,7 +63,7 @@ swarm:
     bzznetworkid: 300
     # LevelDB Store size - by default 4kb * 10^5 == 400 megabytes
     storesize: 100000
-    # List of bootnodes. Example configuration:
+    extraFlags: {}
```

Comment:
Shouldn't this be a list? `extraFlags: []`

Reply:
I don't know honestly. This worked and this is why I merged it.

Is `extraFlags: []` better?

Reply:
When using `{}`, it looks like you need to provide a dictionary.

```yaml
extraFlags: { flag1: "value1", flag2: "value2"  } 
# it's equal to: 
extraFlags:
  flag1: "value1"
  flag2: "value2"
```

When using `[]`, it looks like you need to provide a list. Which is what I think you want to do.

List in yaml:

```yaml
extraFlags: ["--flag1=value1", "--flag2=value2"]
# it's equal to:
extraFlags:
  - "--flag1=value1"
  - "--flag2=value2"
```


Currently it won't hurt, because by default if it's an empty dict `{}` or an empty list `[]` it won't render anything. But it's a bit annoying reading it as a dictionary and then figuring out that you actually have to provide a list.